dtests/src/test/java/io/snappydata/hydra/longRun/longRunTestEmbededMode.conf

```diff
@@ -0,0 +1,134 @@
+hydra.Prms-testRequirement = "Test to verify functionality in embeded mode while running the cluster on long duration with different schemas";
```

Comment:
Test description needs to be more appropriate to what we are testing using this test.
This could be like below:
"Test to verify product behavior when the cluster is up and running for longer duration with continuously performing different tasks through sqlScript, snappyjob and spark App."